tests/test_routers/test_model_performance/test_outcome_count.py

```diff
@@ -24,3 +24,21 @@ def test_get_outcome_count_per_filter(app: FastAPI) -> None:
     data = resp.json()
     metrics = data.pop("countPerFilter")
     assert "label" in metrics and len(metrics["label"]) == 3
+
+
+def test_outcome_count_empty_filters(app: FastAPI):
+    client = TestClient(app)
+    resp = client.get(
+        "/dataset_splits/eval/outcome_count/per_filter?pipeline_index=0&utterance=rrgeth"
```

Comment:
Small suggestion for entertainment:
```suggestion
        "/dataset_splits/eval/outcome_count/per_filter?pipeline_index=0&utterance=yukongold"
```
I really wanted a potato variety that was a QC reference in some way, but the best I could find was [Asterix](https://inspection.canada.ca/plant-health/potatoes/potato-varieties/asterix/eng/1312587385641/1312587385642).

Feel free to ignore this suggestion. 😆